### DISCUSSION TITLE
Display first unquoted text in mobile native notification previews

### DISCUSSION THREAD

**Discussion by nebbles** [community/community#9769] [Mobile]
When receiving a notification about a response written in for example a PR, the body of the notification corresponded to the body of the comment made by the user. 

However if this comment made by the user starts with quoted text, it makes for a pretty confusing notification, and does not provide a preview into the actual response from the user. 

Ideally, notifications would preview the text after the initial quote block to provide a preview of what the responder is commenting with. 

Here’s an example response I received from a commenter:
```
> anyways, so if a request was made with a bad path, or a service that did not exist, the fix/change in code would only occur in the request or the device.

@nebbles yes sounds sensible I'm on it now
```
The notification I received on iOS contained the text preview
```
anyways, so if a request was made with a bad 
```
Whereas preferably I would have been able to preview 
```
@nebbles yes sounds sensible I'm on it now
```

**Comment by AbdulHakim030** [comment]
Yes
